tests/Apis/HelloWorld/HelloWorld.csproj

```diff
@@ -8,4 +8,8 @@
     <FrameworkReference Include="Microsoft.AspNetCore.App" />
   </ItemGroup>
 
+  <ItemGroup>
+    <ProjectReference Include="..\..\..\src\Weikio.ApiFramework.Abstractions\Weikio.ApiFramework.Abstractions.csproj" />
```

Comment:
Should reference SDK, not the Abstractions.

Reply:
Done.